flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -284,6 +284,21 @@ public StreamExecutionEnvironment setParallelism(int parallelism) {
 		return this;
 	}
 
+	/**
+	 * Sets the runtime execution mode for the application (see {@link RuntimeExecutionMode}).
+	 * This is equivalent to setting the "execution.runtime-mode" in your application's
+	 * configuration file.
```

Comment:
I think we should add a note here saying that this is not the recommended way to do it. Users should keep the job itself free of configuration and provided it via the commandline, to stay more flexible. What do you think?

Reply:
Yes, I will add it.